FilesCheck.py

```diff
@@ -1042,7 +1054,7 @@ def check(self, pkg):
         for exe, paths in bindir_exes.items():
             if len(paths) > 1:
                 printWarning(pkg, "duplicate-executable", exe, paths)
-            if exe not in man_basenames:
+            if exe not in man_basenames and not exe.startswith("rc") and len(paths) is not 0:
```

Comment:
The `exe.startswith("rc")` I assume has to do with SUSE's weird `rcFOO` thing?

What is `len(paths) is not 0` for?

Reply:
It is magical check making sure there sctually is executable and it aint empty, the if len(paths) > 1

Reply:
Ah.

Reply:
I'd prefer to see the `exe.startswith("rc")` thing dropped, as it's SUSE-specific.